### PR TITLE
Initial integration of visualization with dataprep backend

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds `cask-datavoyager` node module to be used to integration voyager with cdap dataprep (For reference [cask-datavoyager](https://github.com/ajainarayanan/voyager/)
- Differences between [vega/voyager](https://github.com/vega/voyager) and [cask-datavoyager](https://github.com/ajainarayanan/voyager) is summarized here [Differences](https://github.com/ajainarayanan/voyager/#cask-fork)
- Right now the visualization data is stored as part of the workspace properties
- Any change in the data the visualization is reset as the data/schema has changed for visualization.

JIRA: https://issues.cask.co/browse/CDAP-12592

#### Note:
- Not i18n yet. Getting functionality as of now as language is still not confirmed yet.
- Still testing integration. Will update this PR once the integration is working. Just opened it for code review.

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/DataPrepContentWrapper/index.js [hunk 1]
```diff
@@ -39,24 +43,37 @@ const ViewStore = createStore(
...
-      <div className="btn btn-secondary" onClick={onSwitchChange.bind(null, 'data')}>
-        Data
+    <div className="content-switch">
+      <div
+        className={classnames("switch", {
+          'active': activeTab === 'data'
+        })}
+        onClick={onSwitchChange.bind(null, 'data')}
+      >
+        Data Preparation
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepContentWrapper/index.js]: i18n?

File: cdap-ui/app/cdap/components/DataPrep/DataPrepVisualization/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+export default class DataPrepVisualization extends Component {
+  getInitialState = () => {
+    let {dataprep} = DataPrepStore.getState();
+    return dataprep.data;
+  };
+
+  state = {
+    loading: true,
+    data: {
+      "values": this.getInitialState()
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepVisualization/index.js]: nit: single quote

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by leacd** [comment]
LGTM

**Pull request merged**